packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -242,6 +243,7 @@ beforeAll( async () => {
 	enablePageDialogAccept();
 	observeConsoleLogging();
 	await simulateAdverseConditions();
+	await resetPreferences();
```

Comment:
Playwright tests are passing, but I think this will probably needed to be added to the configuration for Playwright too.

Not sure of the best way. @kevin940726 any advice?

Reply:
I've added that in 8b472ca 👍